codecov.yml

```diff
@@ -0,0 +1,13 @@
+codecov:
+  require_ci_to_pass: true
+  coverage:
+    precision: 2
+    round: down
+    range: "60...100"
+    status:
+      project:
+        default:
+          target: auto
+          threshold: 1%
```

Comment:
so if a PR reduces coverage by 1%, CI will fail?

Reply:
I think so, yes. But it won't block landing the PR.